src/MessagingInput/MessagingInput.tsx

```diff
@@ -60,36 +63,43 @@ const MessagingInputRenderFunction: React.ForwardRefRenderFunction<MessagingInpu
 
   return (
     <FlexBox className={cx(cssClass("Container"), className)} column alignItems={ItemAlign.START}>
-      <label htmlFor={TEXT_FIELD_NAME} className={cssClass("TextFieldLabel")}>
+      <label
+        htmlFor={TEXT_FIELD_NAME}
+        className={cssClass(replyTo ? "TextFieldLabelHidden" : "TextFieldLabel")}
+      >
         {labelText}
       </label>
       <FlexBox className={cssClass("InnerContainer")}>
-        <TextArea
-          ref={textAreaRef}
-          className={cssClass("TextField")}
-          name={TEXT_FIELD_NAME}
-          value={value}
-          onChange={(e) => {
-            onChange(e.target.value);
-          }}
-          onKeyDown={(e) => {
-            // Shift+Enter is a line-break.
-            // Enter alone is an attempt to Send, unless prop is
-            //  explicitly set to have enter be a newline.
-            if (e.key === KeyCode.ENTER && !e.shiftKey && !newlineOnEnter) {
-              e.preventDefault();
-              // If something other than whitespace is in the input area, Send the message.
-              if (value.trim() !== "") {
-                onSubmit(value.trim());
+        <FlexBox column className={cx(cssClass("InnerContainer--Content"))}>
+          {replyTo && <FlexBox className={cx(cssClass("Reply"), className)}>{replyTo}</FlexBox>}
+          <TextArea
+            ref={textAreaRef}
+            className={cssClass("TextField")}
+            name={TEXT_FIELD_NAME}
+            value={value}
+            onChange={(e) => {
+              onChange(e.target.value);
+            }}
+            onKeyDown={(e) => {
+              // Shift+Enter is a line-break.
+              // Enter alone is an attempt to Send, unless prop is
+              //  explicitly set to have enter be a newline.
+              if (e.key === KeyCode.ENTER && !e.shiftKey && !newlineOnEnter) {
+                e.preventDefault();
+                // If something other than whitespace is in the input area, Send the message.
+                if (value.trim() !== "") {
+                  onSubmit(value.trim());
+                }
               }
-            }
-          }}
-          onBlur={onBlur}
-          autoResize
-          // The field starts with `rows + 1` rows, so
-          //  passing in 0 gets us the desired starting height.
-          rows={0}
-        />
+            }}
+            placeholder={replyTo ? labelText : ""}
```

Comment:
We may want to rethink this if we ever want to support a placeholder prop for this input but works for our existing use-case